packages/node/src/node.ts

```diff
@@ -70,7 +70,7 @@ export class Node {
     this.registerIoMiddleware();
   }
 
-  private async init() {
+  private async asyncronouslyConnectRemoteServices(): Promise<Node> {
```

Comment:
This isn't the most accurate either as it's entirely connecting to remote services.

A more accurate name would be `asyncronouslySetupUsingRemoteServices`